src/stores/index.js

```diff
@@ -121,10 +125,13 @@ const Store = Reflux.createStore({
       fetchedConnections: new ConnectionCollection(),
       // Hash for storing unchanged connections for the discard feature
       connections: {},
+      // Message shown when attempting connection.
+      connectingStatus: 'Establishing connection...',
```

Comment:
maybe `connectionStatusText`? Would be cool to have those strings as constants, probably not super important .. feel kinda strange to edit the copy in the middle of the code.